src/main/resources/assets/blocky-bubbles/lang/ru_ru.json

```
@@ -0,0 +1,13 @@
+{
+  "blocky-bubbles.options.culling_awareness.enum.bubble_column": "Пузырьковая колонка (подобная Java)",
+  "blocky-bubbles.options.culling_awareness.enum.non_air": "Невоздушные (паритет)",
+  "blocky-bubbles.options.culling_awareness.enum.none": "Без отбраковки",
+  "blocky-bubbles.options.culling_awareness.name": "Режим отбраковки",
+  "blocky-bubbles.options.culling_awareness.tooltip": "Какие блоки должны вызывать отбраковку соседних граней колонны пузырей.",
+  "blocky-bubbles.options.enable_animations.name": "Включить анимацию",
+  "blocky-bubbles.options.enable_animations.tooltip": "Следует ли анимировать текстуры колонны пузырей (требуется перезагрузка ресурсов).",
+  "blocky-bubbles.options.quality.name": "Колонны пузырей",
+  "blocky-bubbles.options.quality.tooltip": "Контролирует качество колонны пузырей.",
+  "resourcePack.blocky-bubbles:32x_upscale.description": "Основано на §8https://faithfulpack.net",
+  "resourcePack.blocky-bubbles:32x_upscale.name": "Блочные пузыри 32×32"
```

@axialeaa should the resource pack name be translated at all? It contains the mod name, translating it may lead to confusion.

---

Насколько я знаю, в описании ресурспака в игре пишется что-то вроде "(встроен в <имя_мода>)"

---

Как бы да, но в целом в русском языке названия переводить не принято. Улицы, города, названия программ и т.д. пишутся либо транслитом, либо сохраняют оригинальное написание.

---

Кроме фильмов и сериалов, разве что.

---

> @axialeaa should the resource pack name be translated at all? It contains the mod name, translating it may lead to confusion.

no, please keep the name the same if that's alright!

---

> Кроме фильмов и сериалов, разве что.

Да, произведения искусства здесь стоят особняком, их действительно  традиционно переводят.

> no, please keep the name the same if that's alright!
Ok, thanks!

```suggestion
  "resourcePack.blocky-bubbles:32x_upscale.name": "Blocky bubbles 32×32"
```